utils/core/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -129,19 +130,22 @@ internal fun getNetrcAuthentication(contents: String, machine: String): Password
 
     val iterator = lines.joinToString(" ").split(Regex("\\s+")).iterator()
 
-    var machineFound = false
+    var machineFound: String? = null
```

Comment:
How about naming this variable 'machine'?

Reply:
That would conflict with the function parameter of the same name.

Reply:
I missed that, thanks!